packages/rollup-full-node/src/app/web3-rpc-handler.ts

```diff
@@ -284,6 +288,11 @@ export class DefaultWeb3Handler
     return '0x0'
   }
 
+  public async getBalance(): Promise<string> {
+    // Balances are always zero
+    return '0x0'
```

Comment:
@masonforest I presume this is the normal response for a zero balance? It always weirds me out to see an odd number of digits in bytes data